packages/okta-adapter/src/config.ts

```diff
@@ -1778,6 +1779,18 @@ export const DEFAULT_CONFIG: OktaConfig = {
   },
 }
 
+const CLASSIC_ENGINE_UNSUPPORTED_TYPES = [DEVICE_ASSURANCE, AUTHENTICATOR_TYPE_NAME, ACCESS_POLICY_TYPE_NAME,
+  PROFILE_ENROLLMENT_POLICY_TYPE_NAME]
+
+export const getSupportedTypes = ({
+  isClassicOrg,
+  supportedTypes,
+}: {
+  isClassicOrg: boolean
+  supportedTypes: Record<string, string[]>
+}): Record<string, string[]> =>
+  (isClassicOrg ? _.omit(supportedTypes, CLASSIC_ENGINE_UNSUPPORTED_TYPES) : SUPPORTED_TYPES)
```

Comment:
Shouldn't it be the following?
```suggestion
  (isClassicOrg ? _.omit(supportedTypes, CLASSIC_ENGINE_UNSUPPORTED_TYPES) : supportedTypes)
```

Reply:
correct